src/main/drivers/pwm_output.h

```diff
@@ -30,11 +30,27 @@ typedef enum {
     PWM_TYPE_DSHOT600,
     PWM_TYPE_DSHOT300,
     PWM_TYPE_DSHOT150,
+    PWM_TYPE_DSHOT900,
+    PWM_TYPE_DSHOT1200,
     PWM_TYPE_MAX
```

Comment:
I thought about that, and implemented it at first - but then it requires a configurator change at the same time :(